packaging/package.yml/en.md

```diff
@@ -44,44 +42,43 @@ install    : |
 
 ## Keys
 
-Not all fields in `package.yml` are mandatory, but a small selection are. They are listed below. Note that `string(s)` indicates that it is possible to use a `list` of strings, or one single `string`
-
-`dict` refers to a `key : value` split in YAML, and `dict(s)` refers to a list of `dict`s
+Not all fields in `package.yml` are mandatory, but a small selection are. Below is the complete list of the available fields.
 
 ### Mandatory Keys
 
 Key Name | Type | Description
 ---- | ---- | ----
-**name** | `string` | The name of the package. This is also used as the base of all sub-package names. Unless unavoidable, this should match the upstream name
+**name** | `string` | The name of the package. This is also used as the base of all sub-package names. Unless unavoidable, this should match the upstream name.
 **version** | `string` | The version of the currently packaged software. This is taken from the tarball in most cases.
 **release** | `integer` | Specifies the current release number. Updates in the package number are based on this `release` number, *not* the `version` number. As such, to release an update to users, this number must be incremented by one.
 **license** | `string(s)` | Valid upstream license(s). Try to ensure these use [SPDX identifiers](https://spdx.org/licenses/).
-**source** | `dict(s)` | Upstream source location (i.e. tarball), with the valid `sha256sum` as a value
-**component** | `string` | Component / group of packages this package belongs to. Check available components via `eopkg lc`
-**summary** | `string` | Brief package summary, or display name
-**description** | `string` | More extensive description of the software, usually taken from the vendor website
+**source** | `dict(s)` | Upstream source URL (i.e. tarball), with the valid `sha256sum` as a value.<br />Alternatively, the git repository URL prefixed with `git\|` and a git tag or commit hash as a value.
+**component** | `string` | Component / group of packages this package belongs to. Check available components via `eopkg lc`.
```

Comment:
Period at the end should probably remain omitted in case anyone does a lazy copy / pasta.